pyston/converters/extra.py

```diff
@@ -14,8 +14,8 @@ def to_camel_case(snake_str):
 
 
 def to_snake_case(name):
-    s1 = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', name)
-    return re.sub('([a-z0-9])([A-Z])', r'\1_\2', s1).lower()
+    s1 = re.sub('(.)([A-Z])', r'\1_\2', name)
+    return re.sub('([^_])([A-Z])', r'\1_\2', s1).lower()
```

Comment:
Why did you do this? The previous solution was more robust. This will incorectly convert `myHTTPResponse` to `my_h_t_t_p_response` (the original solution converts it correctly to `my_http_response`)

Reply:
because to_camel_case('my_h_t_t_p_response') -> 'myHTTPResponse' 

to_camel_case(to_snake_case('myHTTPResponse')) -> 'myHTTPResponse'

old implementation will be

to_camel_case(to_snake_case('myHTTPResponse')) -> 'myHttpResponse'